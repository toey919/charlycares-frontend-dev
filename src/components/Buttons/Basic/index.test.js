import React from 'react';
import { shallow } from 'enzyme';
import CustomButton, { CButton } from './index';

describe('CustomButon', () => {
  it('Should render styled component', () => {
    let wrapper = shallow(<CustomButton />);
    expect(wrapper).toMatchSnapshot();
  });
  it('Should render component without custom props', () => {
    let wrapper = shallow(<CButton fullWidth />);
    expect(wrapper).toMatchSnapshot();
  });
});
