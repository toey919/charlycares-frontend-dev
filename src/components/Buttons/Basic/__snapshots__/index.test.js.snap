// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButon Should render component without custom props 1`] = `
<Button
  as="button"
/>
`;

exports[`CustomButon Should render styled component 1`] = `
<CButton
  className="sc-bdVaJa lhZWpq"
/>
`;
